service/storage_service.cc

```diff
@@ -356,13 +356,28 @@ future<> storage_service::topology_state_load(cdc::generation_service& cdc_gen_s
             co_await add_normal_node(id, rs);
         }
 
+        std::optional<bool> request_read_new;
+
         for (const auto& [id, rs]: _topology_state_machine._topology.transition_nodes) {
             locator::host_id host_id{id.uuid()};
             auto ip = co_await id2ip(id);
 
             slogger.trace("raft topology: loading topology: raft id={} ip={} node state={} dc={} rack={} tokens state={} tokens={}",
                           id, ip, rs.state, rs.datacenter, rs.rack, rs.ring->state, rs.ring->tokens);
 
+            {
+                const auto read_new = rs.ring.value().state == ring_slice::replication_state::write_both_read_new;
```

Comment:
Don't you need to rebase it on master? Kamil moved state out to be a global per topology, bot per node.

Reply:
done